server/src/pages/shot/view.js

```diff
@@ -386,8 +386,13 @@ class Body extends React.Component {
       favicon = <div style={{backgroundImage: `url("${shot.favicon}")`}} className="favicon" />;
     }
 
-    const shotPageDownload = <Localized id="shotPageDownload"><span className="download-text">Download</span></Localized>;
-
+    const noText = this.props.abTests && this.props.abTests.downloadText
+                           && this.props.abTests.downloadText.value === "no-download-text";
+    const downloadText = [<img src={ this.props.staticLink("/static/img/download-white.svg") } width="20" height="20"/>,
+                         '\u00a0',
+                          <Localized id="shotPageDownload"><span className="download-text">Download</span></Localized>];
+    const downloadNoText = [<img src={ this.props.staticLink("/static/img/download-white.svg") } width="20" height="20"/>,
+                    <span className="download-text"></span>];
```

Comment:
eslint is not happy with these arrays.  Do they need to be arrays?

Reply:
yeah, I got slightly obsessed with finding a way to wedge a non-breaking space into jsx. I'll just put some right-margin on the img instead